scripts/check_frontend_test_coverage.py

```diff
@@ -87,6 +87,7 @@
     'csrf-token.service.ts',
     'current-interaction.service.ts',
     'drag-and-drop-positive-int-editor.component.ts',
+    'edit-thumbnail-modal.component.ts'
```

Comment:
We are not expecting to add a new item to this list, can we add a test for this new component instead?

Reply:
done